mkosi/config.py

```diff
@@ -3603,15 +3604,15 @@ def want_selinux_relabel(config: Config, root: Path, fatal: bool = True) -> Opti
             die(f"SELinux relabel is requested but SELinux file contexts not found in {fc}")
         return None
 
+    # The policy file is named policy.XX where XX is the policy version that indicates what features are
+    # available.
     binpolicydir = root / "etc/selinux" / policy / "policy"
-
-    try:
-        # The policy file is named policy.XX where XX is the policy version that indicates what features are
-        # available. It's not expected for there to be more than one file in this directory.
-        binpolicy = next(binpolicydir.glob("*"))
-    except StopIteration:
+    policies = [p for p in binpolicydir.glob("*") if p.suffix and all(c in string.digits for c in p.suffix[1:])]
```

Comment:
c.isdecimal would have probably been ok here, but the comparison with string.digits is stricter, which is good. A comment to that extent would probably be good so the reasoning is clear to readers.